From 0d1fc45d45573af047770b840402539a5d1ab608 Mon Sep 17 00:00:00 2001
From: Shaofeng Tang <shaofeng.tang@intel.com>
Date: Mon, 19 Aug 2019 10:45:19 +0800
Subject: [PATCH] Set "vendor.hwc.ready" property when boot completed

Set the property "vendor.hwc.ready" as 1 when boot completed
for starting "hdcp" service

Change-Id: I0862e2568507bb51804e62be17d8015f81f69a25
Tests: Work well on KBL-nuc celadon Q
Tracked-On: https://jira.devtools.intel.com/browse/OAM-84587
Signed-off-by: Shaofeng Tang <shaofeng.tang@intel.com>
---
 .../2.1/default/android.hardware.graphics.composer@2.1-service.rc      | 3 +++
 .../2.3/default/android.hardware.graphics.composer@2.3-service.rc      | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/graphics/composer/2.1/default/android.hardware.graphics.composer@2.1-service.rc b/graphics/composer/2.1/default/android.hardware.graphics.composer@2.1-service.rc
index 5a5b51e..4bc606e 100644
--- a/graphics/composer/2.1/default/android.hardware.graphics.composer@2.1-service.rc
+++ b/graphics/composer/2.1/default/android.hardware.graphics.composer@2.1-service.rc
@@ -5,3 +5,6 @@ service vendor.hwcomposer-2-1 /vendor/bin/hw/android.hardware.graphics.composer@
     capabilities SYS_NICE
     onrestart restart surfaceflinger
     writepid /dev/cpuset/system-background/tasks
+
+on property:sys.boot_completed=1
+    setprop vendor.hwc.ready 1
diff --git a/graphics/composer/2.3/default/android.hardware.graphics.composer@2.3-service.rc b/graphics/composer/2.3/default/android.hardware.graphics.composer@2.3-service.rc
index 81ce890..616aeeb 100644
--- a/graphics/composer/2.3/default/android.hardware.graphics.composer@2.3-service.rc
+++ b/graphics/composer/2.3/default/android.hardware.graphics.composer@2.3-service.rc
@@ -5,3 +5,6 @@ service vendor.hwcomposer-2-3 /vendor/bin/hw/android.hardware.graphics.composer@
     capabilities SYS_NICE
     onrestart restart surfaceflinger
     writepid /dev/cpuset/system-background/tasks
+
+on property:sys.boot_completed=1
+    setprop vendor.hwc.ready 1
-- 
1.9.1

